src/components/pages/InstructorHome/InstructorCalender.js

```diff
@@ -30,11 +28,12 @@ const InstructorCalender = ({ dummyData }) => {
 
   function dateCellRender(value) {
     const listData = getListData(value);
+
     return (
       <ul className="events">
         {listData.map(item => (
           <li key={item.content}>
-            <Badge status={item.type} text={item.description} />
+            <Badge status={item.type} text={item.course} />
```

Comment:
Line 36 had the prop "text={item.description}" before being changed to "item.course". Was this a typo or did BE change the schema and table causing this change? Other than that, looks good!

Reply:
Update: Took a look at a PR from earlier this evening and the BE modifying the schema appears to be the reason for the small change in names. Thank you!

Reply:
> Line 36 had the prop "text={item.description}" before being changed to "item.course". Was this a typo or did BE change the schema and table causing this change? Other than that, looks good!

I am not very clear about the two build-in function in this calendar. What I think happened here is : item.course is just the instructor.course_schedule.subject as you can see in line 11 that setCourse update the value of course with itemz.subject. I am still having a lot of questions about this calendar. Right now, for a given case like 8, there will be the same event on the 8th of every month. How to populate the calendar with a specific date is the question.   Even if we transfer the date into a string, and use the string as a case, how are we going to deal with all the different dates. We cannot hard code the case numbers. If anyone has a better understanding about the calendar, please share. Thanks.